pymc_marketing/mmm/__init__.py

```diff
@@ -39,13 +39,25 @@
 )
 from pymc_marketing.mmm.delayed_saturated_mmm import MMM, DelayedSaturatedMMM
 from pymc_marketing.mmm.fourier import MonthlyFourier, YearlyFourier
+from pymc_marketing.mmm.media_transformation import (
+    MediaConfig,
+    MediaConfigs,
```

Comment:
Shall we have more different names? Just as "s" at the end is a typo-rick hehe. For example `MediaConfig` and `MediaConfigList`?

Reply:
Good idea 😆